scripts/start

```diff
@@ -59,13 +59,15 @@ echo
 # Whitelist device IP, hostname and hidden service for CORS
 DEVICE_IP="$(ip -o route get to 8.8.8.8 | sed -n 's/.*src \([0-9.]\+\).*/\1/p')"
 DEVICE_HOSTNAME="$(hostname)"
+DEVICE_DOMAIN=`host  "${DEVICE_HOSTNAME}" | awk '{print $1}' | sed 's/^[^.]*\.//' | head -1`
```

Comment:
I think I've narrowed down my issue to this line. On my machine, the script stops at this line. My hostname is `nephele` so `host $(hostname)` returns an error.

Reply:
After changing this line to `DEVICE_DOMAIN=local` I'm still having issues, regardless of if I connect to the IP or domain.